Source/WebKit/UIProcess/Inspector/win/RemoteWebInspectorUIProxyWin.cpp

```diff
@@ -131,9 +134,48 @@ WebPageProxy* RemoteWebInspectorUIProxy::platformCreateFrontendPageAndWindow()
     return m_webView->page();
 }
 
+void RemoteWebInspectorUIProxy::platformSave(Vector<WebCore::InspectorFrontendClient::SaveData>&& saveDatas, bool forceSaveAs)
+{
+    ASSERT(saveDatas.size() == 1);
+    UNUSED_PARAM(forceSaveAs);
+
+    // Remove custom URI schemes such as "web-inspector://" and also remove the leading "/".
+    URL url { saveDatas[0].url };
+    Vector<WCHAR, MAX_PATH> filePath(url.path().substring(1).toString().wideCharacters());
+
+    OPENFILENAME ofn;
+    memset(&ofn, 0, sizeof(ofn));
+
+    ofn.lStructSize = sizeof(OPENFILENAME);
+    ofn.hwndOwner = m_frontendHandle;
+    ofn.lpstrFile = filePath.data();
+    ofn.nMaxFile = MAX_PATH;
+    ofn.Flags = OFN_PATHMUSTEXIST | OFN_OVERWRITEPROMPT;
+
+    if (GetSaveFileName(&ofn)) {
+        auto fd = openFile(filePath.data(), FileOpenMode::ReadWrite);
+        if (!isHandleValid(fd))
+            return;
+
+        String content { saveDatas[0].content };
+        auto contentSize = content.utf8().length();
+        auto bytesWritten = writeToFile(fd, content.utf8().data(), contentSize);
+        if (bytesWritten == -1 || bytesWritten != contentSize) {
+            auto errMsg = makeString("Error: writeToFile returns ", bytesWritten, ", contentLength = ", content.length());
+            MessageBox(0, errMsg.wideCharacters().data(), L"Export HAR", MB_OK);
```

Comment:
This is user visible error message. It'd be nice if it's better error message. You can take a text error message. https://stackoverflow.com/q/1387064